apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -1685,6 +1762,267 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                 )
             }
 
+            { /* Client Authentication*/ }
+            <Grid.Row columns={ 2 }>
+                <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                    <Divider />
+                    <Divider hidden />
+                </Grid.Column>
+                <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                    <Heading as="h4">
+                        { t("console:develop.features.applications.forms.inboundOIDC.sections" +
+                            ".clientAuthentication.heading") }
+                    </Heading>
+                    <Field
+                        ref={ tokenEndpointAuthMethod }
+                        name="tokenEndpointAuthMethod"
+                        label={
+                            t("console:develop.features.applications.forms.inboundOIDC.sections" +
+                                ".clientAuthentication.fields.authenticationMethod.label")
+                        }
+                        required={ false }
+                        type="dropdown"
+                        disabled={ false }
+                        default={
+                            initialValues?.clientAuthentication ?
+                                initialValues.clientAuthentication.tokenEndpointAuthMethod
+                                : metadata.tokenEndpointAuthMethod.defaultValue
+                        }
+                        placeholder={
+                            t("console:develop.features.applications.forms.inboundOIDC.sections" +
+                                ".clientAuthentication.fields.authenticationMethod.placeholder")
+                        }
+                        listen={ (values: Map<string, FormValue>) => handleAuthMethodChange(values) }
+                        children={ getAllowedList(metadata.tokenEndpointAuthMethod) }
+                        readOnly={ readOnly }
+                    />
+                    <Hint>
+                        { t("console:develop.features.applications.forms.inboundOIDC.sections" +
+                            ".clientAuthentication.fields.authenticationMethod.hint") }
+                    </Hint>
+                </Grid.Column>
+            </Grid.Row>
+
+            { selectedAuthMethod == "private_key_jwt" &&
```

Comment:
```suggestion
            { selectedAuthMethod === "private_key_jwt" &&
```
Consider moving "private_key_jwt" to a constant. Or enum

Reply:
fixed with [be309bd](https://github.com/wso2/identity-apps/pull/4284/commits/be309bdd61f1ecdcf606cdf0bfe85ea775a210a6)